[Testing] WorkshopOptimizerPlugin 0.1.7.0

belzaru17 opened the pull request:
Workshop Optimizer tries to help you manage what to craft in your island sactuary workshops.  It can gather the current supply/demand information by clicking a button, and either exportit to CSV or using it internally.

It looks at combinatins of items and estimated sell value and makes recommendations on what to craft.

philpax commented:
This plugin is now in the plugin review queue; we hope to get back to you soon!

reiichi001 commented:
bleatbot, approve

MidoriKami commented on testing/live/WorkshopOptimizerPlugin/manifest.toml:
```diff
@@ -1,6 +1,6 @@
 [plugin]
 repository = "https://github.com/belzaru17/WorkshopOptimizerPlugin"
```
Change this to:
`repository = "https://github.com/belzaru17/WorkshopOptimizerPlugin.git"`

philpax commented:
Hi there, to help us review this plugin, could you write up a brief description of what it does and what kind of functionality it offers? See https://github.com/goatcorp/DalamudPluginsD17/pull/461 for a good example 🙂

belzaru17 commented:
> Hi there, to help us review this plugin, could you write up a brief description of what it does and what kind of functionality it offers? See #461 for a good example 🙂

Hi Philpax, thanks for the review!  I updated the description to add some more details and added a few screenshots to make it easier to visualize.

Thanks!

philpax commented:
Very sorry about how long this has taken! If you're in the goat place Discord, let me know your Discord username and I'll have you added to the private channels.

belzaru17 commented:
No  worries at all, thanks! Discord username Morgand#3813.
